src/UseMemoContext/index.jsx

```diff
@@ -30,11 +30,7 @@ const UseMemoContext = () => {
         <IconContextProvider icon={icon}>
             <UseMemoContextStyled className="UseMemoContext">
                 <h1>UseMemo Context</h1>
-                <input
-                    placeholder="Name"
-                    value={name}
-                    onChange={(e) => setName(e.target.value)}
-                />
+                <UserInput />
```

Comment:
By moving the input and its state to a child component I am able to achieve the same result as the useMemo solution.